systemd/escape.go

```diff
@@ -0,0 +1,66 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
Two reasons: breaks with go1.3 and for some time this was important for us and the latest version is not in the archive. Both are fixable, however this code was written originally when go1.3 was important for us so its actually more work to remove it again. But I can do that of course (just explaining why its there not arguing one way or the other).